packages/OmegaPrint-Tests.package/TestObjectArrayLiteralElement.class/instance/testWithStatement.st

```diff
@@ -0,0 +1,6 @@
+final expression
+testWithStatement
+
+	self
+		prettify: 'statement'
+		shouldEqual: 'statement'
```

Comment:
Was testen wir hier?
Also was kann hier kaputt gehen?

Reply:
Das ist ja ein aehnlicher Fall wie in vielen anderen Tests, zum Beispiel `TestBinaryMessageSend >> testWithReceiverWithoutMessage`. Es wird getestet, dass wir nicht aus Versehen zu viele Leerzeichen setzen und hier zusaetzlich, dass wir keinen Punkt nach dem letzten Element von `ObjectArrayLiteral`s setzen.